docs/api/app.md

```diff
@@ -27,6 +26,13 @@ Using `ion-app` enables the following behaviors:
 * [Ripple effect](./ripple-effect) when activating buttons on Material Design mode
 * Other tap and focus utilities which make the experience of using an Ionic app feel more native
 
+## Programmatic Focus
+
+As mentioned earlier, the `ion-app` component provides focus utilities for Ionic components with the `ion-focusable` class. The `setFocus` method allows you to programmatically focus an element in response to user actions. However, it should not be used when the element is focused due to a keyboard event, as the focus utility will handle that automatically.
```

Comment:
I'm not quite understanding the last sentence, but that might just be me. I'll leave it up to you to keep it as is or change it.

Reply:
Oh basically when a user hits `tab` on the keyboard to get to the element they shouldn't be overriding that with their own call to `setFocus` since it is already handled. LMK if I can word this better!

Reply:
Oh, then can we have something that distinguishes who is handling it? I read it as the focus utility that comes from `ion-app`: `setFocus`.

```suggestion
As mentioned earlier, the `ion-app` component provides focus utilities for Ionic components with the `ion-focusable` class. The `setFocus` method allows you to programmatically focus an element in response to user actions. However, it should not be used when the element is focused due to a keyboard event, as the keyboard event's focus utility will handle that automatically.
```